fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -52,23 +53,24 @@ object FAPI {
       configJson <- Response.Async.Right(faciaClient.config)
       frontJson <- Response.fromOption(configJson.fronts.get(frontId), NotFound(s"No front found for $frontId"))
       collectionIds = frontJson.collections
-      collectionsJsons <- Response.Async.Right(Future.traverse(collectionIds)(faciaClient.collection))
+      collectionsJsonOptions <- Response.Async.Right(Future.traverse(collectionIds)(faciaClient.collection))
+      collectionsJsons = collectionsJsonOptions.flatten
```

Comment:
Remember that not all collections have collection JSON, so it really does need to be an option.